### PR TITLE
Fix prometheus label when monitoring k8s delete events.

### PR DESCRIPTION
**Pull request by luckyxiaoqiang**

### CODE REVIEW

File: pilot/pkg/serviceregistry/kube/controller.go [hunk 1]
```diff
@@ -197,7 +197,7 @@ func (c *Controller) createCacheHandler(informer cache.SharedIndexInformer, otyp
 				}
 			},
 			DeleteFunc: func(obj interface{}) {
-				k8sEvents.With(prometheus.Labels{"type": otype, "event": "add"}).Add(1)
```
Comment (hzxuzhonghu) [inline pilot/pkg/serviceregistry/kube/controller.go]: I think it is intended. @costinm

Comment (luckyxiaoqiang) [reply pilot/pkg/serviceregistry/kube/controller.go]: > I think it is intended. @costinm

I'm confused. Could you explain more.

**Comment by ymesika** [comment]
Those were already fixed in `release-1.1` by #9853

**Pull request closed without merging**